src/app/post/[postId]/mobile/error.tsx

```
@@ -0,0 +1,40 @@
+
+import { useRouter } from 'next/navigation';
+
+import { experimentPostMobileHeaderLayout } from './components/ExperimentPostMobileHeader/ExperimentPostMobileHeader.css';
+
+import { commonStatusLayout, commonStatusText } from '@/app/common-status.css';
+import RightHeader from '@/components/Header/RightHeader/RightHeader';
+import Icon from '@/components/Icon';
+import MobilePostDetailLayout from '@/components/layout/MobilePostDetailLayout/MobilePostDetailLayout';
+import { colors } from '@/styles/colors';
+
+export default function Error() {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)